openmeetings-web/src/main/java/org/apache/openmeetings/web/room/raw-video.js

```diff
@@ -287,7 +287,7 @@ var Video = (function() {
 			});
 		}
 		v.parent().find('.ui-dialog-titlebar-close').remove();
-		v.parent().find('.ui-dialog-titlebar').append(OmUtil.tmpl('#video-button-bar'));
+		v.parent().append(OmUtil.tmpl('#video-button-bar'));
```

Comment:
Move icons one level on top of the ui-dialog-titlebar. ui-dialog-titlebar is the element jquery-ui makes dragable. 
Like this it looks exactly the same but doesn't have the issue, so that on Mobile you can still click the icons AND drag the dialog.